components/raftstore-v2/src/raft/storage.rs

```diff
@@ -184,6 +183,21 @@ impl<ER: RaftEngine> Storage<ER> {
     pub fn is_initialized(&self) -> bool {
         self.region_state.get_tablet_index() != 0
     }
+
+    #[inline]
+    pub fn applied_index(&self) -> u64 {
```

Comment:
This doesn't seem to be a useful abstraction.

Reply:
it's used in store_applied_index() without exposing the internal fields of Storage.

Reply:
`store_applied_index` is another useless abstraction. To query applied_index, just use `self.storage().entry_storage().applied_index()`. And there should be at most two places that needs to query the information.